### DISCUSSION TITLE
Can anyone tell me if the only solution to this problem is to buy more data?

### DISCUSSION THREAD

**Discussion by LuscasD** [community/community#69716] [Programming Help]
### Body

Hi guys, I have a problem that I'm participating in someone else's project but when I try to clone the repository I get this message (This
repository is over its data quota.Account responsible for LFS bandwidth should purchase more data packs to restore acces)

### Guidelines

- [ ] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by LuscasD** [comment]
Cloning into 'D:\JogosQueEuFaço\Estagio\Mat\Mathris'...
remote: Enumerating objects: 2306, done.        
remote: Counting objects: 100% (773/773), done.        
remote: Compressing objects: 100% (335/335), done.        
remote: Total 2306 (delta 518), reused 654 (delta 427), pack-reused 1533        
Receiving objects: 100% (2306/2306), 286.75 MiB | 20.03 MiB/s, done.
Resolving deltas: 100% (1396/1396), done.
Updating files: 100% (1312/1312), done.
Downloading Assets/Firebase/Plugins/x86_64/FirebaseCppApp-11_2_0.bundle (160 MB)
Error downloading object: Assets/Firebase/Plugins/x86_64/FirebaseCppApp-11_2_0.bundle (ce9cceb): Smudge error: Error downloading Assets/Firebase/Plugins/x86_64/FirebaseCppApp-11_2_0.bundle (ce9ccebd84ce0c55263dfa34868d36eb972272d77d91d24454188cfa25b0f673): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.

Errors logged to 'D:\JogosQueEuFaço\Estagio\Mat\Mathris\.git\lfs\logs\20231010T142838.2951993.log'.
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs filter-process' failed
fatal: Assets/Firebase/Plugins/x86_64/FirebaseCppApp-11_2_0.bundle: smudge filter lfs failed
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'


Would you like to retry cloning ?     // Full Log

**Reply by jjrh92** [reply]
Found 2 things that maybe can help.

https://stackoverflow.com/questions/62905325/this-repository-is-over-its-data-quota-account-responsible-for-lfs-bandwidth-sh
https://docs.github.com/en/billing/managing-billing-for-git-large-file-storage/upgrading-git-large-file-storage
